### PR TITLE
Public Libraries

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for the [Utah Public Libraries](https://gis.utah.gov/products/sgid/society/public-libraries/) layer. Let me know what you think. Thank you!

### CODE REVIEW

File: metadata/society/public_libraries.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+### What does the dataset represent?
+
+Each point feature in this dataset includes the library name, type, address, city, and county. Features also include contact information for the library, such as a phone number, PO Box, or website where applicable. We also included library hours of operation where the information has been provided.
+
+### How was the dataset created?
+
+UGRC derived this dataset using tabular data from the Utah State Library Division. We maintain this dataset and update it as new information becomes available.
+
+<!--- This is my guesstimate on how this dataset was created. Do we have further information on how this dataset was created or how it is maintained? --->
```
Comment (ZachBeck) [inline metadata/society/public_libraries.md:35]: Yeah, IIRC we were given a table that I geocoded.

**Review by ZachBeck — COMMENTED** [review]
Looks good

File: metadata/society/public_libraries.md (hunk 1 above)
Comment (steveoh) [reply metadata/society/public_libraries.md:35]: Is that table something that we request or they provide us on some sort of schedule?

Comment (miriamseely) [reply metadata/society/public_libraries.md]: Spoke with Zach about this, the library division sends us updated data as needed, sounded like there wasn't a specific schedule.

Comment (steveoh) [reply metadata/society/public_libraries.md]: :bell: this seems to have fallen into the cracks

Comment (steveoh) [reply metadata/society/public_libraries.md]: The answer to the question. It sounds like you got the answer offline and didn't share it. Thanks for closing the loop.

**Pull request merged**